### PR TITLE
Added concept endpoint to server

### LINKED ISSUE #959: Add active concept endpoint to server
The list of of active concepts ( full qualified name, ...) should be accessable via the server

### PR DESCRIPTION
**Pull request by JAmmermann-DLR**
Added:
- `ServerConcept ` POJO class wrapping a model concept (as these are generated and we can't easlily add annotation there)
- Concept endpoint to the `ModelAccessResource `that has a QueryParam to get only the active concepts or all concepts

Closes #959

### CODE REVIEW

File: de.dlr.sc.virsat.server/src/de/dlr/sc/virsat/server/resources/ModelAccessResource.java [hunk 1]
```diff
@@ -439,6 +450,62 @@ public Response deleteSei(@PathParam("seiUuid") @ApiParam(value = "Uuid of the S
...
+					concepts = repository.getActiveConcepts();
+				} else {
+					// Get all concepts from the registry
+					concepts = new ArrayList<Concept>();
+					IExtensionRegistry registry = Platform.getExtensionRegistry();
+					String extensionPoint = ActiveConceptConfigurationElement.EXTENSION_POINT_ID_CONCEPT;
+					
+					// Get the concept configuration elements
+					IConfigurationElement[] configurationElementsArray = registry.getConfigurationElementsFor(extensionPoint);
+					List<IConfigurationElement> configurationElements = (List<IConfigurationElement>) Arrays.asList(configurationElementsArray);
```
Comment (SaMuellerDLR) [inline de.dlr.sc.virsat.server/src/de/dlr/sc/virsat/server/resources/ModelAccessResource.java:489]: Why do we need to create this list here? I see below we just use it to loop over the elements.
I think we can also just for each loop over the array directly.

**Review by SaMuellerDLR — CHANGES_REQUESTED** [review]
Great work! Only minor code quality question!

**Review by franzTobiasDLR — APPROVED** [review]

File: de.dlr.sc.virsat.server/src/de/dlr/sc/virsat/server/resources/ModelAccessResource.java (hunk 1 above)
Comment (JAmmermann-DLR) [reply de.dlr.sc.virsat.server/src/de/dlr/sc/virsat/server/resources/ModelAccessResource.java:489]: Yeah makes no sense, I will change it

**Review by SaMuellerDLR — APPROVED** [review]
Approved!

**Pull request merged**